### PR TITLE
{bio}[foss/2019b] GROMACS v2019.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2614.swalot.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ed62e555025374bcb9658af5e5338afb for a full test report.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# Copyright:: Copyright 2012-2016 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+# Authors::
+# * Wiktor Jurkowski <wiktor.jurkowski@gmail.com>
+# * Fotis Georgatos <fotis@cern.ch>
+# * George Tsouloupas <g.tsouloupas@cyi.ac.cy>
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Adam Huffman <adam.huffman@crick.ac.uk>
+# License::   MIT/GPL
+# 2019.3 version
```
Comment (zao) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-foss-2019b.eb]: This comment has the wrong version

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3135.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/25d9748fc338c5a97b98471ff7caaa91 for a full test report.

File: easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-foss-2019b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/GROMACS/GROMACS-2019.4-foss-2019b.eb]: fixed in 00e8f78

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (1 easyconfigs in this PR)
node3301.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 2.7.5
See https://gist.github.com/8dddf8cbeed43e0c56e2bac82d486dd8 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**